docs/_static/js/rst_links_fix.js

```diff
@@ -0,0 +1,4 @@
+window.onload = function() {
```

Comment:
Did you check the js to work on built documentations (or Read The Doc) ?

Reply:
I checked it locally on my machine - all seems to be OK.
On Travis it's done by emulation of this js function.

Reply:
Since it's JQuery, it should work with all browsers. In theory...

Reply:
okay.

> In theory...

yeah, in theory 😅